rllib/core/learner/scaling_config.py

```diff
@@ -13,7 +14,8 @@ class LearnerGroupScalingConfig:
             training will run on a single CPU.
         num_gpus_per_worker: The number of GPUs to allocate per worker. If
             num_workers=0, any number greater than 0 will run the training on a single
-            GPU. A value of zero will run the training on a single CPU.
+            GPU. A value of zero will run the training on a single CPU. Fractional
```

Comment:
fractional values aren't really supported. Fractional gpu ussage will cause cuda aync errors AFAICT

Reply:
Thanks, will fix. ...

Reply:
done

Reply:
Actually, there is one test, where we do set num_gpus_per_worker to 0.5:

In `test_learner_group.py`:
```
LOCAL_SCALING_CONFIGS = {
    "local-cpu": LearnerGroupScalingConfig(num_workers=0, num_gpus_per_worker=0),
    "local-gpu": LearnerGroupScalingConfig(num_workers=0, num_gpus_per_worker=0.5),
}
```

Reply:
I think that's why I fixed the typehint, but yeah, it makes sense that torch/tf DDP wouldn't like it. I fixed the comment and said it's not allowed due to xyz.